Automattic/wp-calypso
# DIFM: Remove retired Glen and Antin themes

#### Changes proposed in this Pull Request

* Glen and Antin themes are retired. These themes need to be removed from the static DIFM list as well.

#### Testing instructions



* Go to `/start/do-it-for-me`.
* On the design picker step, confirm that the Glen and Antin themes are not present.



Related to 1201454687238941-as-1201933307121197/f
